contrib/capitalone_dataprofiler_expectations/capitalone_dataprofiler_expectations/tests/rule_based_profiler/domain_builder/test_data_profiler_column_domain_builder.py

```diff
@@ -19,7 +19,6 @@
 from great_expectations.data_context import FileDataContext
 from great_expectations.rule_based_profiler.domain_builder import DomainBuilder
 from great_expectations.rule_based_profiler.parameter_container import (
-    VARIABLES_KEY,
```

Comment:
@alexjeilert Now with the simplification of the `DomainBuilder`, are any of these tests duplicates, which could be deleted?  Thanks.